dds_web/api/errors.py

```diff
@@ -226,14 +215,24 @@ def __init__(self, message="No method found in request."):
         general_logger.warning(message)
 
 
-class PublicKeyNotFoundError(exceptions.HTTPException):
-    """Public key not found in database"""
+class KeyNotFoundError(exceptions.HTTPException):
+    """Key not found in database."""
 
     def __init__(self, project, message="No key found for current project"):
         self.message = f"{message}: {project}"
         super().__init__(self.message)
 
-        general_logger.warning(self.message)
+
+class PublicKeyNotFoundError(KeyNotFoundError):
```

Comment:
Do we need more types of exceptions?

Reply:
No I understood what you meant :) It's changed to `KeyNotFoundError` instead of `PublicKey...`, `PrivateKey...` and `S3Keys...`

Reply:
They were used after the refactoring in the other branches (the next PRs) but I agree that it may be a bit too picky to have so many different custom ones